lib/runner/extract-runnable-items.js

```diff
@@ -186,6 +186,11 @@ var sdk = require('postman-collection'),
 
         items = findItemsOrGroups(collection, entrypointLookup, true);
 
+        // Sorting the items by the order in which they appear as folder arguments.
+        items && items.members && items.members.sort(function (a, b) {
+            return entrypoints.indexOf(a.name) - entrypoints.indexOf(b.name);
```

Comment:
Entry points will not always be just names. They can have id's in which case the indexOf will fail and just return `-1`.